src/hotspot/os/windows/os_windows.cpp

```diff
@@ -2552,6 +2542,27 @@ LONG WINAPI topLevelExceptionFilter(struct _EXCEPTION_POINTERS* exceptionInfo) {
         return EXCEPTION_CONTINUE_SEARCH;
       }
 
+#ifdef _M_ARM64
+      // Unsafe memory access
```

Comment:
I'm not at all clear why this unsafe memory access handling is for Aarch64 only?

Reply:
Hum, this was already part of [JDK-8250810](https://github.com/openjdk/jdk/commit/a4eaf9536c272862b5ec856bf263679be09bddc9) / [JDK-8248817](https://github.com/openjdk/jdk/commit/257809d7440e87ac595d03b6c9a98d8f457f314c)  (see a couple lines below in this PR) without the `ifdef` for Aarch64, but I messed up rebasing on top of it. I removed it now, thanks for catching!